### PR TITLE
G Suite: Update Email Comparison page for Google Workspace

### PR DESCRIPTION
**Pull request by stephanethomas**
This pull request updates the `Email Comparison` page in order to support Google Workspace:

Before | After
------ | -----
before | after

This also enables Google Workspace by default for development environments. Finally, it removes some code that would manage customer data, and that is no longer used.

#### Testing instructions

1. Run `git checkout update/email-comparison-page` and start your server, or open a [live branch](https://calypso.live/?branch=update/email-comparison-page)
2. Log into a WordPress.com account with a domain
3. Open the [`Domains` page](http://calypso.localhost:3000/domains/manage)
4. Click the `Add` button for that domain
5. Assert that you are presented with a page that matches the screenshot above

### CODE REVIEW

File: client/my-sites/email/email-providers-comparison/index.jsx [hunk 1]
```diff
@@ -221,20 +222,42 @@ class EmailProvidersComparison extends React.Component {
...
+			translate( '30GB storage' ),
+			translate( 'Email, calendars, and contacts' ),
+			translate( 'Video calls, docs, spreadsheets, and more' ),
+		];
+		let buttonLabel = translate( 'Add G Suite' );
+
+		if ( config.isEnabled( 'google-workspace-migration' ) ) {
+			title = getGoogleMailServiceFamily();
+			description = translate(
+				'The best way to create, communicate, and collaborate. An integrated workspace that is simple and easy to use.'
```
Comment (stephanethomas) [inline client/my-sites/email/email-providers-comparison/index.jsx:235]: The new copy and logo were taken from the partner brand guidelines.

File: client/my-sites/email/email-providers-comparison/index.jsx [hunk 2]
```diff
@@ -221,19 +222,39 @@ class EmailProvidersComparison extends React.Component {
...
+				title={ title }
+				description={ description }
+				image={ { path: logo } }
 				features={ [
 					translate( 'Annual billing' ),
 					translate( 'Send and receive from your custom domain' ),
 					translate( '30GB storage' ),
 					translate( 'Email, calendars, and contacts' ),
-					translate( 'Video calls, Docs, spreadsheets, and more' ),
+					translate( 'Video calls, docs, spreadsheets, and more' ),
```
Comment (a8ci18n) [inline client/my-sites/email/email-providers-comparison/index.jsx:256]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/email/email-providers-comparison/index.jsx (hunk 1 above)
Comment (olaseni) [reply client/my-sites/email/email-providers-comparison/index.jsx:235]: A reference/link would have been nice here, for posterity.

File: config/development.json [hunk 3]
```diff
@@ -69,7 +69,7 @@
 		"gdpr-banner": false,
 		"google-my-business": true,
 		"google-drive": true,
-		"google-workspace-migration": false,
+		"google-workspace-migration": true,
```
Comment (olaseni) [inline config/development.json:72]: You may need to add a version of this to `wpcalypso.json` in case someone uses the live branch, like I am.

**Review by olaseni — APPROVED** [review]
A couple of non-blocking notes, but LGTM!

File: client/my-sites/email/email-providers-comparison/index.jsx (hunk 1 above)
Comment (stephanethomas) [reply client/my-sites/email/email-providers-comparison/index.jsx:235]: That's a good point. Unfortunately those guidelines are not publicly accessible, which is whyI didn't link to them here.

File: config/development.json (hunk 3 above)
Comment (stephanethomas) [reply config/development.json:72]: Thank you. That was my initial intent but I couldn't find any documentation about which configuration file calypso.live is using. I thought modifying `development.json` would do the trick but then I forgot about testing that change.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5427978

Thank you @stephanethomas for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5427978).

**Pull request merged**